src/app/functions/server/featureFlags.ts

```diff
@@ -34,5 +46,19 @@ export async function getFlag(name: string): Promise<FeatureFlag | undefined> {
     owner: data.owner,
   };
 
-  return flag;
+  if (!flag.isEnabled) {
+    console.warn("Flag is not enabled:", flag.name);
```

Comment:
Aren't the `console` messages (both this and the others in this file) going to be very noisy in our logs?

Reply:
Possibly, I'm a bit worried about how to debug what's wrong if things aren't working for someone... I'll think about it some more, maybe there's a better way to do this.

Reply:
Isn't `mozlog` intended to allow us to log messages with different levels that only show up on particular environments?

Reply:
Yes it is, `mozlog` is in a bit of an odd state right now but I think there's been some work recently on bringing it up-to-date. I am not sure it does very good structured logging for GCP or anything else that improves much over the `console.*` levels (`info/warning/error` etc), which are handled OK by the GCP logging tools.